### PR TITLE
Added portworx volume support for jenkins master

### PR DESCRIPTION
**Pull request by tusharrautOO7**

### CODE REVIEW

File: universe/config.json [hunk 1]
```diff
@@ -43,15 +43,34 @@
...
                 "pinned-hostname": {
                     "description": "An optional DC/OS agent hostname to run this Jenkins instance on (e.g. 10.0.0.1).",
                     "type": "string"
                 }
-            }
+            },
+            "required": [
+                "portworx-volume-name",
+                "portworx-volume-size",
+                "portworx-volume-replication-factor"
```
Comment (piyush-nimbalkar) [inline universe/config.json:72]: No need to keep them required as such. Anyway from UI they will be necessary but from the CLI we should be able to apply without size and repl as portworx uses defaults for those.

File: universe/marathon.json.mustache [hunk 2]
```diff
@@ -41,38 +41,25 @@
       {"port": {{networking.agent-port}}, "protocol": "tcp", "name": "agent"}
   ],
   "container": {
-        "type": "{{advanced.containerizer}}",
```
Comment (piyush-nimbalkar) [inline universe/marathon.json.mustache]: Why are changing this to use only MESOS container type?

File: universe/marathon.json.mustache [hunk 3]
```diff
@@ -41,38 +41,25 @@
...
+        "type": "MESOS",
         "docker": {
-         {{#advanced.docker-image}}
```
Comment (piyush-nimbalkar) [inline universe/marathon.json.mustache]: Why are we removing this?

Comment (tusharrautOO7) [reply universe/marathon.json.mustache]: Reverted.

File: universe/marathon.json.mustache (hunk 2 above)
Comment (tusharrautOO7) [reply universe/marathon.json.mustache]: Reverted this changes. as default containerizer was set to Docker

File: universe/config.json [hunk 4]
```diff
@@ -171,7 +190,7 @@
                     "description": "The containerizer used to deploy the Jenkins master.",
                     "type": "string",
                     "enum": ["MESOS", "DOCKER"],
-                    "default": "DOCKER"
```
Comment (piyush-nimbalkar) [inline universe/config.json:174]: Why not leave it to DOCKER if that is what they set it to?
Does it not work with Docker and portworx changes?

**Review by piyush-nimbalkar — APPROVED** [review]

File: universe/config.json (hunk 4 above)
Comment (tusharrautOO7) [reply universe/config.json:174]: portworx volume changes are not working with containerizer DOCKER, so default set it to MESOS

**Pull request merged**